soc/xlnx/zynq7000/xc7zxxxs/Kconfig.soc

```diff
@@ -9,9 +9,6 @@
 config SOC_SERIES_XC7ZXXXS
 	bool
 	select SOC_FAMILY_XILINX_ZYNQ7000
-	help
-	  Enable support for the Xilinx Zynq-7000S (XC7ZxxxS)
-	  SoC series (single core ARM Cortex-A9).
```

Comment:
also looks pretty useful to me -- `SOC_SERIES_XC7ZXXXS` is quite cryptic.

Reply:
Likewise, you select a SOC and it selects the series for you. And you get a much better representation by looking at the soc.yml file